File: SNS.Web/src/app/jobs/companies/contracts/company-summary.dto.ts
import { CompanyRole } from '../../enums/company-role.enum';

export interface CompanySummaryDto {
    id: string;
    name: string;
    industry: string;
    websiteUrl: string | null;
    logoUrl: string | null;
    createdAt: string;
    myRole: CompanyRole | null;
}

File: backend/SNS.Application/Jobs/Companies/Contracts/CompanySummaryDto.cs
using SNS.Domain.Jobs.Enums;

namespace SNS.Application.Jobs.Companies.Contracts;

public sealed record CompanySummaryDto(
    Guid Id,
    string Name,
    string Industry,
    string? WebsiteUrl,
    string? LogoUrl,
    DateTime CreatedAt,
    int ActiveJobsCount,
    CompanyRole? CurrentUserRole
);
